doc/runtimes/nodejs.md

```diff
@@ -46,8 +46,10 @@ The following process is applied for each build.
 2. Run custom build command if specified by `CUSTOM_BUILD_COMMAND`. This will skip running any package install commands 
    (`npm install` or `yarn install`) that Oryx originally included.
 3. Run custom run build command if specified by `RUN_BUILD_COMMAND`. This will not skip package install commands.
-4. Run `npm install` without any flags, which includes npm `preinstall` and
-   `postinstall` scripts and also installs devDependencies.
+4. Run `npm install`, which includes npm `preinstall` and `postinstall` scripts and also installs devDependencies.
+   The `--unsafe-perm` flag is added to the NPM install command, as well as the `--production` flag if some production
```

Comment:
I actually removed the unsafe-perm flag in this PR, as it isn't needed now that the permissions are updated: https://github.com/microsoft/Oryx/pull/1759#pullrequestreview-1225116576

Reply:
Ah good call, thanks. I'll update this.

Reply:
Can't believe I forgot this one. Here is the update.